style.css

```diff
@@ -514,3 +513,59 @@ li {
     width: 385px;
   }
 }
+
```

Comment:
- [ ] Kindly update the color of the close button of the mobile menu to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/215491600-829c0cd1-bf2d-46f5-97f9-964338abfc33.png) | ![image](https://user-images.githubusercontent.com/24703171/215491671-fb605884-9f52-4a8d-b1c1-42a7d0310ecb.png) |